src/xdocs/config_javadoc.xml

```diff
@@ -3690,6 +3690,31 @@ public class Test extends Exception {
   public String InvalidFunction(){
     return "";
   }
+}
+        </source>
+        <p>
+              Example of non permitted empty javadoc for Inline Summary Javadoc.
+        </p>
+        <source>
+public class Test extends Exception {
+  /**
+   * {@summary  }
+   */
+  public String InvalidFunctionOne(){ // violation
+    return "";
+  }
+
+  /**
+   * {@summary &lt;p&gt; &lt;p/&gt;}
+   */
+  public String InvalidFunctionTwo(){ // violation
+    return "";
+  }
+
+  /**
+   * {@summary &lt;p&gt;This is summary for validFunctionThree.&lt;p/&gt;}
```

Comment:
@AkMo3 For my own understanding, what is the proper way to create summaries like this for javadocs? Should they be wrapped in a `<p>` tag like this or should it start with pure text and go from there?

Reply:
Sir, there is no specific limitation about the tags can be used in [How to Write Doc Comments for the Javadoc Tool](https://www.oracle.com/in/technical-resources/articles/java/javadoc-tool.html) and depends on the user`s choice of how they want them to be customized.